arguments.py

```diff
@@ -0,0 +1,8 @@
+import argparse
+
+parser = argparse.ArgumentParser('SWE1R scripts ')
+parser.add_argument('--out', description='directory where the output files are written', default='/tmp/swep1r')
```

Comment:
As we will probably have a lot of windows users, I'd like to get rid of the "/tmp/" folder altogether.
It would be fine with me if it somehow chose the platforms temp folder or output to a subfolder of the current directory by default.

(the scripts also lack the ability to generate the output folder if it doesn't exist yet, but that's probably a seperate issue)